config/interface_elements/households/households_housing_stock.yml

```diff
@@ -172,106 +172,69 @@ groups:
 
   - header: share_useful_demand_houses
     type: slider
+    combination_method:
+      weighted_average:   
+        - sum:
+          - present_number_of_apartments_before_1945
+          - present_number_of_apartments_1945_1964
+          - present_number_of_apartments_1965_1984
+          - present_number_of_apartments_1985_2004
+          - present_number_of_apartments_2005_present
+          - present_number_of_detached_houses_before_1945
+          - present_number_of_detached_houses_1945_1964
+          - present_number_of_detached_houses_1965_1984
+          - present_number_of_detached_houses_1985_2004
+          - present_number_of_detached_houses_2005_present
+          - present_number_of_semi_detached_houses_before_1945
+          - present_number_of_semi_detached_houses_1945_1964
+          - present_number_of_semi_detached_houses_1965_1984
+          - present_number_of_semi_detached_houses_1985_2004
+          - present_number_of_semi_detached_houses_2005_present
+          - present_number_of_terraced_houses_before_1945
+          - present_number_of_terraced_houses_1945_1964
+          - present_number_of_terraced_houses_1965_1984
+          - present_number_of_terraced_houses_1985_2004
+          - present_number_of_terraced_houses_2005_present
```

Comment:
Would a comment concerning the combination method be useful here: explaining how we use the total number of residences?